prow/plugins/plugins.go

```diff
@@ -86,13 +87,27 @@ func (pa *PluginAgent) Load(path string) error {
 	if err := yaml.Unmarshal(b, &np); err != nil {
 		return err
 	}
+	// Check that there are no plugins that we don't know about.
 	for _, v := range np {
 		for _, p := range v {
 			if _, ok := allPlugins[p]; !ok {
 				return fmt.Errorf("unknown plugin: %s", p)
 			}
 		}
 	}
+	// Check that there are no duplicates.
+	for k, v := range np {
+		if strings.Contains(k, "/") {
+			org := strings.Split(k, "/")[0]
+			for _, p1 := range v {
+				for _, p2 := range np[org] {
```

Comment:
Probably doesn't matter but maybe we should switch to using sets.String from k8s.io/pkg/util/sets.